ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,21 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- floor_status:
```

Comment:
See comment on `EPM`

Reply:
`status` should be reserved for binary states. I suggest something like `floor_position_mode`